migrations/versions/0740bd1e69ef_create_tables.py

```diff
@@ -729,26 +728,13 @@ def upgrade():
 
 
 def downgrade():
-    # TODO: need to fix these autogenerated commands as they are not working for some reason
-    # I think this might help: https://alembic.sqlalchemy.org/en/latest/ops.html#alembic.operations.Operations.drop_constraint
     # ### commands auto generated by Alembic - please adjust! ###
```

Comment:
Is this 'adjusted' now? in which case change comment

Reply:
No I don't know if this has been tested yet. It's highly unlikely that we would downgrade this far back and delete all the tables so probably worth the risk.